app/views/handlers/question.py

```diff
@@ -228,9 +228,11 @@ def capture_dependent_sections_for_list(self, list_name):
         section_ids = self._schema.get_section_ids_dependent_on_list(list_name)
         section_ids.append(self.current_location.section_id)
 
-        section_keys_to_add = self.questionnaire_store_updater.started_section_keys(
-            section_ids=section_ids
-        )
+        # Only add sections which are repeated sections for this list, or the section in which this list is collected
+        # Prevents list item progresses being added as dependants
+        section_keys_to_add = [section_key
+                               for section_key in self.questionnaire_store_updater.started_section_keys(section_ids=section_ids)
+                               if not (section_key[0] == self.current_location.section_id and section_key[1])]
```

Comment:
I'm a bit confused by this change. Why are we doing this?. The comment didn't clear up much.

Reply:
`started_section_keys(section_ids=section_ids)` with the current section id will return the list items' progresses as they are keyed by this section and their list item id.

So this filter is to ensure they aren't captured as dependant sections (unless they are supposed to be? In this impl I don't think they are supposed to be). I can reword to try and be more clear + I've refactored the function to be more elegant anyway so might make more sense.

Reply:
I'm not sure I still follow here, would be good to have a chat about it.

Reply:
No action needed following tech session convo 24/05/23